servicetalk-http-api/src/main/java/io/servicetalk/http/api/MultiAddressHttpClientBuilder.java

```diff
@@ -114,8 +117,7 @@
     @Override
     public MultiAddressHttpClientBuilder<U, R> appendClientFilter(final Predicate<StreamingHttpRequest> predicate,
                                                                   final StreamingHttpClientFilterFactory factory) {
-        super.appendClientFilter(predicate, factory);
-        return this;
```

Comment:
can you clarify why the `cast` approach is used here?

Reply:
Just wanted to make it consistent to the approach we use in other places:
1. https://github.com/apple/servicetalk/blob/05c28f8339441494f12defece5770dee700d1702/servicetalk-http-api/src/main/java/io/servicetalk/http/api/MultiAddressHttpClientBuilder.java#L84
2. https://github.com/apple/servicetalk/blob/05c28f8339441494f12defece5770dee700d1702/servicetalk-http-api/src/main/java/io/servicetalk/http/api/BaseSingleAddressHttpClientBuilder.java#L57
3. https://github.com/apple/servicetalk/blob/05c28f8339441494f12defece5770dee700d1702/servicetalk-http-api/src/main/java/io/servicetalk/http/api/PartitionedHttpClientBuilder.java#L74-L75